BlueHarvest/DataStore.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DataStore.swift
+//  BlueHarvest
+//
+//  Created by David Nutter on 3/19/17.
+//  Copyright © 2017 David Nutter. All rights reserved.
+//
+
+import Foundation
+
+class DataStore {
+    
+    static let shared = DataStore()
+    
+    private static let profiles = "profiles"
+    private let backgroundQueue = DispatchQueue(label: "com.dnutter.datastore")
+
+    private static var filePath : String {
+        let manager = FileManager.default
+        guard let url = manager.urls(for: .documentDirectory, in: .userDomainMask).first else {
+            assertionFailure()
+            return ""
```

Comment:
It appears that FileManager.urls(for:,in:) will always return at least one valid url. Perhaps it is safe to use the forced unwrapped optional here and get rid of the guard let altogether